### PR TITLE
feat: php class codebyte example

### PR DESCRIPTION
**Pull request by davialano**
### Description

Created a codebyte example.

### Issue Solved 
#5188 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/php/concepts/classes/classes.md [hunk 1]
```diff
@@ -108,3 +108,65 @@ The `instanceof` operator returns `true` when the operand on the left belongs to
   // Output: It is a Car!
 ?>
 ```
+
+## Codebyte Example
+
+Now, with the above understanding, we can create any class together, defining attributes and methods for various contexts. In the following code snippet, we create the class Library, and you can experiment with the `__construct()` method by creating global attributes, methods and then test all the implementation:
```
Comment (SaviDahegaonkar) [inline content/php/concepts/classes/classes.md:114]:
```suggestion
Now, with the above understanding, we can create any class together, defining attributes and methods for various contexts. In the following code snippet, we create the class Library, and you can experiment with the `__construct()` method by creating global attributes, and methods and then test all the implementation:
```

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @davialano ,
Your entry LGTM!. I now approve it for a second review.

File: content/php/concepts/classes/classes.md [hunk 2]
```diff
@@ -108,3 +108,65 @@ The `instanceof` operator returns `true` when the operand on the left belongs to
   // Output: It is a Car!
 ?>
 ```
+
+## Codebyte Example
+
+Now, with the above understanding, we can create any class together, defining attributes and methods for various contexts. In the following code snippet, we create the class Library, and you can experiment with the `__construct()` method by creating global attributes, and methods and then test all the implementation:
```
Comment (avdhoottt) [inline content/php/concepts/classes/classes.md:114]:
```suggestion
The following code defines a `Library` class with attributes for library details, methods to display library information and register books with associated data. Try executing it:
```

File: content/php/concepts/classes/classes.md [hunk 3]
```diff
@@ -108,3 +108,65 @@ The `instanceof` operator returns `true` when the operand on the left belongs to
...
+
+    return print_r($portfolio);
+  }
+}
+
+$libraryOne = new Library('The British Library', 1, 'United Kingdom');
+
+$libraryOne->getInfoLibrary();
+
+$libraryOne->registerBook('J.R.R. Tolkien', 'The Silmarillion', 1977);
```
Comment (avdhoottt) [inline content/php/concepts/classes/classes.md:169]:
```suggestion
<?php
class Library
{
  private $libraryName;
  private $ranking;
  private $country;

  // The constructor to initialize the class
  public function __construct(string $libraryName, int $ranking, string $country)
  {
    $this->libraryName = $libraryName;
    $this->ranking = $ranking;
    $this->country = $country;
  }

  // Method to get information about the library
  public function getInfoLibrary()
  {
    $info = $this->libraryName . ' is the number ' . $this->ranking . ' in the world ranking. Located in ' . $this->country . '.<br><br>';
    echo $info;
  }

  // Method to register a new book
  public function registerBook(string $author, string $bookTitle, int $publishYear)
  {
    $portfolio = [
      'author' => $author,
      'bookTitle' => $bookTitle,
      'publishYear' => $publishYear
    ];
    
    print_r($portfolio); // print array for debugging or return $portfolio for further use
  }
}

// Create a new Library instance and call methods
$libraryOne = new Library('The British Library', 1, 'United Kingdom');
$libraryOne->getInfoLibrary();
$libraryOne->registerBook('J.R.R. Tolkien', 'The Silmarillion', 1977);
?>
```

File: content/php/concepts/classes/classes.md [hunk 4]
```diff
@@ -108,3 +108,52 @@ The `instanceof` operator returns `true` when the operand on the left belongs to
...
+}
+
+// Create a new Library instance and call methods
+$libraryOne = new Library('The British Library', 1, 'United Kingdom');
+$libraryOne->getInfoLibrary();
+$libraryOne->registerBook('J.R.R. Tolkien', 'The Silmarillion', 1977);
+?>
+```
+
+Now, it's your turn!
```
Comment (avdhoottt) [inline content/php/concepts/classes/classes.md:159]:
```suggestion
```

**Pull request merged**